scipy/stats/setup.py

```diff
@@ -28,9 +29,24 @@ def configuration(parent_package='',top_path=None):
         sources=['mvn.pyf','mvndst.f'],
     )
 
+    # add BiasedUrn module
+    config.add_data_files('biasedurn.pxd')
+    config.add_extension(
```

Comment:
Need add c++11/14 compiler flags, that's probably where the errors are coming from

Reply:
Done. Thanks!